store/useShapeDiverStoreViewer.ts

```diff
@@ -111,7 +112,24 @@ export const useShapeDiverStoreViewer = create<IShapeDiverStoreViewer>()(devtool
 
 		let session: ISessionApi|undefined = undefined;
 		try {
-			session = await createSession(dto);
+			try {
+				session = await createSession(dto);
+			}
+			catch (e: any) {
+				if (isViewerGeometryBackendResponseError(e) && 
+					e.geometryBackendErrorType === ShapeDiverResponseErrorType.REQUEST_VALIDATION_ERROR &&
+					e.message.startsWith("Invalid parameter") && e.message.includes("'context'")) 
+				{
+					console.warn("Session creation failed due to invalid or missing 'context' parameter. Retrying without 'context' parameter.");
+
+					const dtoWithoutContext: SessionCreateDto = {...dto, initialParameterValues: {...dto.initialParameterValues}};
+					delete dtoWithoutContext.initialParameterValues!["context"];
```

Comment:
@snabela I'm not fully getting this line. If you assign the dto.initialParameterValues in the line before that, why is there a need to remove the context here? It shouldn't be there in this object anyway, right?

Reply:
@MajorMeerkatThe3rd the line above just deep copies `dto.initialParameterValues`, but it doesn't remove `context`. Please note that `context` is already part of the `dto` object when this function is called. Does this answer your concern?

Reply:
@snabela Ah got it, I thought that was one level above that